src/main/webapp/community.html

```diff
@@ -46,7 +47,17 @@
   </head>
   <body onload="buildUI()">
 
-    <div id="nav-bar"></div>
+    <!-- <div id="nav-bar"></div> -->
```

Comment:
Let's delete this comment - if we need it back, we can always check the commit history.